neuralpp/symbolic/basic_expression.py

```diff
@@ -289,14 +347,22 @@ def basic_add_operation(type_) -> BasicAbelianOperation:
     return BasicAbelianOperation(operator.add, BasicConstant(0, type_), operator.neg)
 
 
-def BasicSummation(type_: type, index: Variable, constraint: Context, body: Expression) -> BasicQuantifierExpression:
+def basic_summation(
```

Comment:
Renamed this and BasicIntegral to follow the python convention of naming functions